### PR TITLE
feat: [#410] Remove gookit/color, use pterm/pterm instead

### PR DESCRIPTION
**Pull request by kkumar-gcc**
Closes https://github.com/goravel/goravel/issues/410

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: config/application.go [hunk 1]
```diff
@@ -28,22 +28,23 @@ func NewApplication(envPath string) *Application {
...
 	appKey := app.Env("APP_KEY")
 	if !support.IsKeyGenerateCommand {
 		if appKey == nil {
-			color.Redln("Please initialize APP_KEY first.")
-			color.Println("Create a .env file and run command: go run . artisan key:generate")
-			color.Println("Or set a system variable: APP_KEY={32-bit number} go run .")
+			color.Red().Println("Please initialize APP_KEY first.")
+			// TODO: Add more information about how to generate APP_KEY.
+			//color.Println("Create a .env file and run command: go run . artisan key:generate")
+			//color.Println("Or set a system variable: APP_KEY={32-bit number} go run .")
```
Comment (hwbrzzl) [inline config/application.go:42]: When finish the TODO?

File: support/color/color.go [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+func White() Printer {
+	return New(FgWhite)
+}
+
+func Gray() Printer {
+	return New(FgGray)
+}
+
+func Normal() Printer {
```
Comment (hwbrzzl) [inline support/color/color.go:86]:
```suggestion
func Default() Printer {
```

File: support/color/color.go [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+var (
+	info    = pterm.Info
+	warn    = pterm.Warning
+	err     = pterm.Error
+	debug   = pterm.Debug
+	success = pterm.Success
+)
+
+type Printer interface {
```
Comment (hwbrzzl) [inline support/color/color.go:42]: Move to the `contracts` folder?

File: support/color/color_test.go [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+			assert.Equal(t, expected, color.Sprint(testString))
+			assert.Equal(t, expectedLn, color.Sprintln(testString))
+			assert.Equal(t, expected, color.Sprintf(format, testString))
+			assert.Equal(t, expectedLn, color.Sprintfln(format, testString))
+
+			//assert.Equal(t, expected, captureOutput(func() { print(color.Print(testString)) }))
+			//assert.Equal(t, expectedLn, captureOutput(func() { println(color.Println(testString)) }))
+			//assert.Equal(t, expected, captureOutput(func() { print(color.Printf(format, testString)) }))
+			//assert.Equal(t, expectedLn, captureOutput(func() { println(color.Printfln(format, testString)) }))
```
Comment (hwbrzzl) [inline support/color/color_test.go:51]: Remove them if not needed.

Comment (kkumar-gcc) [reply support/color/color_test.go]: Sorry i was writing it, finished it now.

File: config/application.go (hunk 1 above)
Comment (kkumar-gcc) [reply config/application.go:42]: mistake

File: support/color/color.go (hunk 3 above)
Comment (kkumar-gcc) [reply support/color/color.go]: Okay

File: console/console/key_generate_command.go [hunk 5]
```diff
@@ -44,34 +43,34 @@ func (receiver *KeyGenerateCommand) Extend() command.Extend {
...
-		color.Yellowln("*     Application In Production!     *")
-		color.Yellowln("**************************************")
-		color.Println(color.New(color.Green).Sprintf("Do you really wish to run this command? (yes/no) ") + "[" + color.New(color.Yellow).Sprintf("no") + "]" + ":")
+		color.Yellow().Println("**************************************")
+		color.Yellow().Println("*     Application In Production!     *")
+		color.Yellow().Printfln("**************************************")
+		color.Default().Println(color.Green().Sprintf("Do you really wish to run this command? (yes/no) ") + "[" + color.Green().Sprintf("no") + "]" + ":")
 
 		var result string
 		_, err := fmt.Scanln(&result)
```
Comment (hwbrzzl) [inline console/console/key_generate_command.go:52]: Can we use `ctx.Confirm` directly?

Comment (kkumar-gcc) [reply console/console/key_generate_command.go:52]: It will be fixed in next PR. (make commands support -f flag)

**Pull request merged**